_pages/about.html

```diff
@@ -150,10 +151,9 @@ <h2 class="section__title">Wanna code with us?</h2>
   <div id="map" class="locations__map"></div>
   <div class="locations__text-box">
     <p class="locations__text">
-      Do you fancy to code with us, drink a coffee (or two), play table soccer or have a yoga class together?<br />
-      No matter where you live now, we'll help you to relocate and settle in.
+      Do you fancy to code with us, drink a coffee (or two), play table soccer or have a yoga class together?
```

Comment:
@AliceStatler Is yoga still a thing? Does it take place?

Reply:
@dkluever22 No, they don't. How sad! I will change this here.